docs/product-assembly_case-management-getting-started/src/main/asciidoc/case-management-designing-a-case-proc.adoc

```diff
@@ -89,7 +102,14 @@ In the IT Orders case, the status of `Milestone 1: Order placed` depends on the
 +
 image::milestone-evaluation-condition.png[Milestone evaluation condition configuration]
 +
-* For more information about the milestones in the IT Orders case, see <<case-management-itorders-milestones-ref-{context}>>. 
+* For more information about the milestones in the IT Orders case, see 
+ifdef::case-management-getting-started[]
+<<case-management-itorders-milestones-ref-{context}>>.
+endif::[]
+ifdef::case-management-design[]
+{URL_GETTING_STARTED_CASES}[_{GETTING_STARTED_CASES}_].
+endif::[]
+
 * To learn more about milestones, see <<case-management-milestones-con-{context}>>.
 
```

Comment:
Above, you have a step "Add a milestone to the case ..." followed by a single sub-step. Not sure if that was intentional, but typically single steps should be avoided, consolidated, or not numbered. Should you just make it the next step? I don't know enough about the process to say. Or it can just be an indented part of the super-step or something.

And you have the "For more info" bits as bullets. I'd nix the bullets and just left align with the numbered indentation.

Also, the milestones cross-refs are not showing up in the rendered output because they're not set up right. I assume you meant to use ifevals?